docs/investigation/2022-06-20-cfg-via-crds.md

```diff
@@ -0,0 +1,581 @@
+# Configure BotKube via CRs
+
+Created on 2022-06-20 by Mateusz Szostok ([@mszostok](https://github.com/mszostok))
+
+| Status      |
+|-------------|
+| `POSTPONED` |
```

Comment:
If this is an investigation, maybe it should be deleted? Not sure what about `Created on...`, but I would rather delete that too. But it's up to you 🙂 

```suggestion
```

Once we know we'll go in CRDs, this investigation could be converted to an proposal 👍